lib_spi/wscript

```diff
@@ -0,0 +1,6 @@
+def use_module(bld):
+    bld.env['XCC_FLAGS_spi_async.xc'] = bld.env.XCC_FLAGS + [
```

Comment:
Should this now be setting `MODULE_XCC_FLAGS` with the contents of `XCC_FLAGS` plus the library specified flags?

Reply:
This one is ok. This is setting file specific flags for spi_async.xc.